sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -186,7 +190,11 @@ def load(*args, **kwargs) -> "AzureAppConfigurationProvider":
     provider = _buildprovider(
         connection_string, endpoint, credential, uses_key_vault="UsesKeyVault" in headers, **kwargs
     )
-    provider._load_all(headers=headers)
+    try:
+        provider._load_all(headers=headers)
+    except Exception as e:
```

Comment:
In case the discussion in JS provider repo got missed: https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/35#discussion_r1427260328